tests/docs/test_prepare_prior_versions.py

```diff
@@ -133,7 +133,7 @@ def test__use_relative_path_for_imports_substitution_path_starting_with_forwards
 
 @pytest.mark.unit
 def test__prepend_version_info_to_name_for_md_relative_links():
-    contents = """For more information on pre-configuring a Checkpoint with a Batch Request and Expectation Suite, please see [our guides on Checkpoints](../../../../docs/guides/validation/index.md#checkpoints)."""
+    contents = """For more information on pre-configuring a Checkpoint with a Batch Request and Expectation Suite, see [Manage Checkpoints](../../../../docs/guides/validation/checkpoints/checkpoint_lp.md)."""
```

Comment:
You'll also need to update this in the test:

```
expected_contents = """For more information on pre-configuring a Checkpoint with a Batch Request and Expectation Suite, please see [our guides on Checkpoints](../../../../docs/0.16.16/guides/validation/#checkpoints)."""
```

Reply:
This code also needs to be updated: https://github.com/great-expectations/great_expectations/blob/60b4487cdc6f95ac0779e980664ef220e4080f7f/docs/prepare_prior_versions.py#L382 since it looks for index.md specifically

Screenshot of failing test:
<img width="1183" alt="Screenshot 2023-07-17 at 10 50 51 AM" src="https://github.com/great-expectations/great_expectations/assets/2117313/e380f5a3-cfe9-4133-ae2b-9b851baae946">